tests/integ/test_packaging.py

```diff
@@ -305,7 +305,7 @@ def test_add_unsupported_requirements_twice_should_not_fail_for_same_requirement
 
 
 @pytest.mark.skipif(
-    IS_IN_STORED_PROC or IS_WINDOWS,
+    IS_IN_STORED_PROC,
```

Comment:
The IS_WINDOWS safeguard will be added back for all tests (see https://github.com/snowflakedb/snowpark-python/pull/947).

They are temporarily undone here to test the changes on Windows.

Reply:
I also remember in experimental PR, you mentioned that this might not work well with windows. Do you want to remove that comment now?

Reply:
No, these failures are flaky. The pre-commit succeeding on this PR does not imply that it won't happen again.

I feel the answer here is to not support the package auto-upload feature on Windows and only look into changing that (and removing the Windows safeguards) if there is a request from users to do so. What do you think?

Will merge the experimental PR after this one so that the tests will not run on Windows.